packages/page-settings/src/General.tsx

```diff
@@ -13,6 +14,7 @@ import uiSettings, { SettingsStruct } from '@polkadot/ui-settings';
 import { useTranslation } from './translate';
 import { createIdenticon, createOption, save, saveAndReload } from './util';
 import SelectUrl from './SelectUrl';
+import NetworkSpecs from './modals/NetworkSpecs';
```

Comment:
Probably not in here. But under Settings we now that the `Extensions` tab which deals with metadata injection. (Wrongly named for this use-case as well) We can just drop the modal completely and always display the QR there.